### PR TITLE
Kit Logistics: Add SR2022 specific information

### PR DESCRIPTION
**Pull request by trickeydan**

### CODE REVIEW

**Review by PeterJCLaw — COMMENTED** [review]
Seems good to me, but I'm not able to comment on correctness.

**Review by PeterJCLaw — APPROVED** [review]
Approving for speed, though would still be good to get more eyes on this from the @srobo/kit-team

File: docs/kit/logistics/transport/couriers.md [hunk 1]
```diff
@@ -19,3 +15,33 @@ Kit data for couriers:
...
+* Lithium Polymer batteries must be individually packaged and sealed
+  * This is to prevent shorts.
+  * The batteries do not need to be in a LiPo bag, although we would strongly recommend that they are.
+* A LiPo warning sticker must be affixed to the outside of the box.
+* The LiPos must be secured such that they will not move during shipping.
+* The box must be flat-sided and "square" to qualify for low shipping rates
+
+### Additional Packing Requirements for S5-UPS
+
+* LiPos must be individually inside jiffy bags, then inside the LiPo bag.
```
Comment (RealOrangeOne) [inline docs/kit/logistics/transport/couriers.md:42]: The below packaging info mentions that batteries "must" be in their own jiffy bags, however that being in this section here implies it's supplier (S5) specific? True or not, I wonder if we should just mandate it anyway?

File: docs/kit/logistics/transport/couriers.md [hunk 2]
```diff
@@ -19,3 +15,33 @@ Kit data for couriers:
...
+### Additional Packing Requirements for UPS
+
+* Lithium Polymer batteries must be individually packaged and sealed
+  * This is to prevent shorts.
+  * The batteries do not need to be in a LiPo bag, although we would strongly recommend that they are.
+* A LiPo warning sticker must be affixed to the outside of the box.
+* The LiPos must be secured such that they will not move during shipping.
+* The box must be flat-sided and "square" to qualify for low shipping rates
+
+### Additional Packing Requirements for S5-UPS
```
Comment (RealOrangeOne) [inline docs/kit/logistics/transport/couriers.md:40]: I believe the S5-UPS restrictions are a superset of the UPS ones?

```suggestion
#### Additional Packing Requirements for S5-UPS
```

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**